src/Accordion/Accordion.js

```diff
@@ -31,6 +34,7 @@ class Accordion extends React.Component {
   static defaultProps = {
     items: [],
     multiple: false,
+    skin: skins.standard,
```

Comment:
Doesn't renders properly in storybook (storybook issue):
```suggestion
    skin: 'standard',
```